app/views/provider/admin/backend_apis/show.html.slim

```diff
@@ -0,0 +1,33 @@
+h1 Backend Overview
+
+section.Section
+  .SettingsBox
+    = link_to 'edit', edit_provider_admin_backend_api_path(@backend_api), class: 'SettingsBox-toggle'
+    dl.SettingsBox-summary.u-dl data-state="open"
+      dt.u-dl-term Name
+      dd.u-dl-definition == @backend_api.name
+      dt.u-dl-term System Name
+      dd.u-dl-definition == @backend_api.system_name
+      - if (description = @backend_api.description.presence)
+        dt.u-dl-term Description
+        dd.u-dl-definition == description
+      dt.u-dl-term Private Base URL
+      dd.u-dl-definition == @backend_api.private_endpoint
+
+section class="service-widget"
+  div.content-service
+
+    section[name="settings"]
+      h2 Products Using this Backend API
+      ul.application-plans
+        - list_items_or_empty @backend_api.services, 'No Product currently using this Backend API' do |service|
+          li.item
+            = link_to service.name, admin_service_path(service)
+
+    section[name="activity"]
+      h2 Methods & Mapping Rules
+      ul.application-plans
+        li.item
+          = link_to pluralize(@backend_api.method_metrics&.size, 'method'), provider_admin_backend_api_metrics_path(@backend_api)
```

Comment:
I don't understand this `&`. It is not considering that `@backend_api` is `nil` (or anything other than an instance of `BackendApi` in this case) because then it would raise a `NoMethodError` when doing `method_metrics`... so given that it assumes it responds to `method_metrics`, why it things that it can be `nil`? it always returns an `ActiveRecord_AssociationRelation` which always responds to size... even if you do `BackendApi.new.method_metrics.size` the result is `0` 😄

Reply:
That was from before when those methods were being delegated to the backend api's first service, which could be `nil`. No longer needed since https://github.com/3scale/porta/pull/1052 and https://github.com/3scale/porta/pull/1053.